### PR TITLE
Clean of various functions to aid legibility

### PR DESCRIPTION
**Pull request by dlaydon**
mostly InfectSweep, TreatSweep and some IncubRecoverySweep and DoPlaceClose.

### CODE REVIEW

**Review by NeilFerguson — APPROVED** [review]

File: src/Sweep.cpp [hunk 1]
```diff
@@ -290,666 +290,590 @@ void InfectSweep(double t, int run) // added run number as argument in order to
...
-						s = ranf_mt(tn);
+						// pick RandomNum between 0 and 1
+						double RandomNum = ranf_mt(ThreadNum);
 						// generate l using InvCDF of selected cell and random integer between 0 and 1024
-						int l = c->InvCDF[(int)floor(s * 1024)];
-						// loop over c->cum_trans array until find a value >= random number s
-						while (c->cum_trans[l] < s) l++;
+						int l = ThisCell->InvCDF[(int)floor(RandomNum * 1024)];
+						// loop over ThisCell->cum_trans array until find a value >= RandomNum
+						while (ThisCell->cum_trans[l] < RandomNum) l++;
```
Comment (weshinsley) [inline src/Sweep.cpp:764]: Might as well zap this "l" at some point.
The two flags (KeepSearching...) and DigiContactTrace_ThisPersonNow could be bools

File: src/Sweep.cpp [hunk 2]
```diff
@@ -961,102 +885,101 @@ void InfectSweep(double t, int run) // added run number as argument in order to
...
-		for (int i = 0; i < P.NumHolidays; i++)
+		for (int HolidayNumber = 0; HolidayNumber < P.NumHolidays; HolidayNumber++)
 		{
-			ht = P.HolidayStartTime[i] + P.HolidaysStartDay_SimTime;
+			ht = P.HolidayStartTime[HolidayNumber] + P.HolidaysStartDay_SimTime;
 			if ((t + P.ModelTimeStep >= ht) && (t < ht))
 			{
-//				Files::xfprintf_stderr("Holiday %i t=%lg\n", i, t);
-				for (int j = 0; j < P.PlaceTypeNum; j++)
+//				Files::xfprintf_stderr("Holiday %HolidayNumber t=%lg\n", HolidayNumber, t);
```
[the remark this answers is not in the document]
Comment (weshinsley) [reply src/Sweep.cpp:897]: That said, it should probably be %d in any case

File: src/Sweep.cpp [hunk 3]
```diff
@@ -961,102 +885,101 @@ void InfectSweep(double t, int run) // added run number as argument in order to
...
 		{
-			Cell* c = CellLookup[b]; //// find (pointer-to) cell.
-			for (int j = ((int)c->L - 1); j >= 0; j--) //// loop backwards over latently infected people, hence it starts from L - 1 and goes to zero. Runs backwards because of pointer swapping?
-				if (ts >= Hosts[c->latent[j]].latent_time) //// if now after time at which person became infectious (latent_time a slight misnomer).
-					DoIncub(c->latent[j], ts, tn); //// move infected person from latently infected (L) to infectious (I), but not symptomatic
-			//StateT[tn].n_queue[0] = StateT[tn].n_queue[1] = 0;
-			for (int j = c->I - 1; j >= 0; j--) ///// loop backwards over Infectious people. Runs backwards because of pointer swapping?
+			Cell* ThisCell = CellLookup[CellIndex]; //// find (pointer-to) ThisCell.
+			for (int LatentPerson = ((int)ThisCell->L - 1); LatentPerson >= 0; LatentPerson--) //// loop backwards over latently infected people, hence it starts from L - 1 and goes to zero. Runs backwards because of pointer swapping?
+				if (TimeStepNow == Hosts[ThisCell->latent[LatentPerson]].latent_time) //// if now after time at which person became infectious (latent_time a slight misnomer).
```
Comment (weshinsley) [inline src/Sweep.cpp:928]: So >= has become == 

Fine as long as TimeStepNow increases by 1 each time - so this func gets called every timestesp, the RHS is effectively constant and integer - all of which I think are true?

I prefer >= even so - it feels like there must have been for some reason in the original...

File: src/Sweep.cpp [hunk 4]
```diff
@@ -961,102 +885,101 @@ void InfectSweep(double t, int run) // added run number as argument in order to
...
+					DoIncub(ThisCell->latent[LatentPerson], TimeStepNow, ThreadNum); //// move infected person from latently infected (L) to infectious (I), but not symptomatic
+
+			for (int InfeciousPersonIndexWithinCell = ThisCell->I - 1; InfeciousPersonIndexWithinCell >= 0; InfeciousPersonIndexWithinCell--) ///// loop backwards over Infectious people. Runs backwards because of pointer swapping?
```
Comment (weshinsley) [inline src/Sweep.cpp:931]: Infecious -> Infectious typos

File: src/Sweep.cpp [hunk 5]
```diff
@@ -961,102 +885,101 @@ void InfectSweep(double t, int run) // added run number as argument in order to
...
+							DoDeath_FromCriticalorSARIorILI	(InfectiousPersonIndex, ThreadNum);
 						else
-							DoRecover_FromSeverity(ci, tn);
+							DoRecover_FromSeverity(InfectiousPersonIndex, ThreadNum);
 					}
 				}
 
 				//Adding code to assign recovery or death when leaving the infectious class: ggilani - 22/10/14
-				if (ts >= si->recovery_or_death_time)
+				if (TimeStepNow == InfectiousPerson->recovery_or_death_time)
```
Comment (weshinsley) [inline src/Sweep.cpp:956]: Similar comment to eralier on >= changing into ==

Ok if you're sure that this gets called every timestep, and the right-hand-sides don't change so the event gets missed.

File: src/Sweep.cpp [hunk 6]
```diff
@@ -1404,46 +1327,40 @@ int TreatSweep(double t)
...
-		if (P.DoPerCapitaTriggers)
-			global_trig = (int)floor(((double)State.trigDetectedCases) * P.GlobalIncThreshPop / ((double)P.PopSize));
-		else
-			global_trig = State.trigDetectedCases;
+		if (P.DoPerCapitaTriggers)	global_trig = (int)floor(((double)State.trigDetectedCases) * P.GlobalIncThreshPop / ((double)P.PopSize));
+		else						global_trig = State.trigDetectedCases;
 	}
-	else
-		global_trig = 0;
+	else							global_trig = 0;
```
Comment (weshinsley) [inline src/Sweep.cpp:1351]: int global_trig = P.DoGlobalTrigger ? (P.DoPerCapitaTriggers ? (int)floor ...  : State.trigDetectedCases) : 0;

if you liike,,,

Or less extreme, int global_trig = 0; on 1345, and delete 1351

File: src/Sweep.cpp [hunk 7]
```diff
@@ -1613,155 +1554,171 @@ int TreatSweep(double t)
...
-				if ((Mcells[mcellnum].vacc == TreatStat::ToBeTreated) && (ts >= Mcells[mcellnum].vacc_start_time))
+				if ((Mcells[mcellnum].vacc == TreatStat::ToBeTreated) && (TimeStepNow >= Mcells[mcellnum].vacc_start_time))
 				{
-					TreatFlag = 1;
-					Mcells[mcellnum].vacc_trig = 0;
+					TreatFlag					= 1;
+					Mcells[mcellnum].vacc		= TreatStat::Treated;
+					Mcells[mcellnum].vacc_trig	= 0;
 					//if(State.cumVG+P.NumThreads*Mcells[mcellnum].n<P.VaccMaxCourses) //changed to VG - commented this out for now, we'll add everyone to queues and deal with the number of doses available in the vaccination function
 					{
```
Comment (weshinsley) [inline src/Sweep.cpp:1565]: Tempting to delete/comment the brace on 1565, and wherever it closes,

File: src/Sweep.cpp [hunk 8]
```diff
@@ -1613,155 +1554,171 @@ int TreatSweep(double t)
...
-						int ad2 = ad / P.VaccAdminUnitDivisor;
+						int ad2 = AdminUnit / P.VaccAdminUnitDivisor;
 						do
 						{
-							if (P.is_in_bounds(min))
+							// depending on characteristics of the nearby Microcells (starting with this microcell), alter their TreatStat variables (start time, TreatStat etc.)
+							if (P.is_in_bounds(Nearby_mcellposition))
 							{
+								bool VaccinateThisMicroCell = 0;
```
Comment (weshinsley) [inline src/Sweep.cpp:1609]: = false;

File: src/Sweep.cpp [hunk 9]
```diff
@@ -1795,115 +1754,139 @@ int TreatSweep(double t)
...
 					{
-						int ad2 = ad / P.MoveRestrAdminUnitDivisor;
+						int ad2 = AdminUnit / P.MoveRestrAdminUnitDivisor;
 						do
 						{
-							if (P.is_in_bounds(min))
+							// depending on characteristics of the nearby Microcells (starting with this microcell), alter their TreatStat variables (start time, TreatStat etc.)
+							if (P.is_in_bounds(Nearby_mcellposition))
 							{
+								bool TreatThisMicroCell = 0;
```
Comment (weshinsley) [inline src/Sweep.cpp:1799]: = false; perhaps

**Pull request merged**